templates/base.html

```diff
@@ -13,12 +13,12 @@
   </script>
 </head>
 <body>
-    {% with errors = get_flashed_messages(category_filter=["account_err"]) %}
+    {% with errors = get_flashed_messages(category_filter=["message"]) %}
         {% if errors %}
             {%- for msg in errors %}
-                <div class="alert alert-danger">
+                <div class="alert alert-{{ msg.split(';', 1)[0] }}">
                     <a href="#" class="close" data-dismiss="alert" aria-label="close">&times;</a>
-                    {{ msg }}
+                    {{ msg.split(';', 1)[1] }}
```

Comment:
I'm not going to make this a blocking issue, but I'm not a fan of this string-splitting in the templates.

It seems to me that better use of the label (you could have `warning` and `error` in addition to `message` could keep you from needing to use a string for two purposes.